### PR TITLE
Changes for scaling policy test

### PR DESCRIPTION
**Pull request by sobhanagit**
Checklist
---------

JIRA Ticket -

Checklist (Please mark Yes/No/NA for each of the questions below) -

- I have provided clear and detailed explanation of changes made in the proposed PR : No
- I have compiled the code successfully my laptop : No
- I have written examples to demonstrates the usage of the newly implemented APIs : No
- I have incremented the release version : No
- I have run the Intellij Code Analysis on newly added code and fixed all (valid) reported issues : No

OR

- I have taken approval from scrum master or project manager for skipping any of the items above : No

### CODE REVIEW

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    GIGABYTES("gigabytes"),
+    TERABYTES("terabytes"),
+    BITS("bits"),
+    KILOBITS("kilobits"),
+    MEGABITS("megabits"),
+    GIGABITS("gigabits"),
+    TERABITS("terabits"),
+    PERCENT("percent"),
+    COUNT("count"),
+    BYTESORSECOND("bytes/second"),
```
Comment (anuragsharma-123) [inline src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java:23]: Can we change these to BYTESPERSECOND and similar for all following fields?

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricSource.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+        for (AwsMetricSource credit : AwsMetricSource.values()) {
+            if (name.equalsIgnoreCase(credit.name)) {
+                retVal = credit;
+                break;
+            }
+        }
+
+        if (retVal == null) {
+            LOGGER.error("Tried to create cpuCredits enum for: " + name + ", but we don't support such type ");
```
Comment (anuragsharma-123) [inline src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricSource.java:29]: Logger message 'cpuCredits' not correct.

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+        for (AwsMetricAlarmUnit credit : AwsMetricAlarmUnit.values()) {
+            if (name.equalsIgnoreCase(credit.name)) {
+                retVal = credit;
+                break;
+            }
+        }
+
+        if (retVal == null) {
+            LOGGER.error("Tried to create cpuCredits enum for: " + name + ", but we don't support such type ");
```
Comment (anuragsharma-123) [inline src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java:58]: Logger message not correct.

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricStatistics.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+        for (AwsMetricStatistics credit : AwsMetricStatistics.values()) {
+            if (name.equalsIgnoreCase(credit.name)) {
+                retVal = credit;
+                break;
+            }
+        }
+
+        if (retVal == null) {
+            LOGGER.error("Tried to create cpuCredits enum for: " + name + ", but we don't support such type ");
```
Comment (anuragsharma-123) [inline src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricStatistics.java:34]: Logger message not correct

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java (hunk 1 above)
Comment (sobhanagit) [reply src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java:23]: resolved

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java (hunk 3 above)
Comment (sobhanagit) [reply src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java:58]: REsolved

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricStatistics.java (hunk 4 above)
Comment (sobhanagit) [reply src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricStatistics.java:34]: resolved

File: src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricSource.java (hunk 2 above)
Comment (sobhanagit) [reply src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricSource.java:29]: Resolved

**Review by anuragsharma-123 — APPROVED** [review]

**Pull request merged**